Meadowlark-js/backends/meadowlark-mongodb-backend/src/index.ts

```diff
@@ -21,3 +21,6 @@ export function initializeDocumentStore(): DocumentStorePlugin {
     securityMiddleware,
   };
 }
+
+// Accessible for system testing - this may turn into a generic setup/teardown datastore interface
+export { getNewClient, getCollection, resetSharedClient } from './repository/Db';
```

Comment:
Exporting these for testing as a first cut. I think we'll want turn these into generic setup/teardown-for-testing functions so that the tests will work with any backend.

Reply:
Yes, something like that. Maybe just a "testSetup" and "testTeardown" interface. Something to get the tests out of the business of importing datastore-specific stuff.